README.rst

```diff
@@ -64,7 +64,7 @@ Contributing
 ------------
 
 We love contributions!  folium is open source, built on open source,
-and we'd love to have you hang out in our community.
+and we had love to have you hang out in our community.
```

Comment:
That is not really an "improved docs" as it only expands the contraction. Also, that contraction expands to "would." Closing this. Hope it is not hacktoberfest spam.